data/brands/shop/truck_repair.json

```diff
@@ -125,6 +125,75 @@
         "name": "Volvo Trucks",
         "shop": "truck_repair"
       }
+    },
+    {
+      "displayName": "Ashok Leyland",
+      "id": "blahblah",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["ashoke"],
+      "tags": {
+        "brand": "Ashok Leyland",
+        "brand:wikidata": "Q726193",
+        "name": "Ashok Leyland",
+        "shop": "truck_repair"
+      }
+    },
+    {
+      "displayName": "BharatBenz",
+      "id": "blahblah",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "brand": "BharatBenz",
+        "brand:wikidata": "Q1107991",
+        "name": "BharatBenz",
+        "shop": "truck_repair"
+      }
+    },
+    {
+      "displayName": "Eicher Motors",
+      "id": "blahblah",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "brand": "Eicher Motors",
+        "brand:wikidata": "Q1301821",
+        "name": "Eicher Motors",
+        "shop": "truck_repair"
+      }
+    },
+    {
+      "displayName": "Mahindra Truck and Bus",
+      "id": "blahblah",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "brand": "Mahindra Truck and Bus",
+        "brand:wikidata": "Q848059",
+        "name": "Mahindra Truck and Bus",
+        "shop": "truck_repair"
+      }
+    },
+    {
+      "displayName": "SML Mahindra (formerly SML Isuzu)",
+      "id": "blahblah",
+      "locationSet": {"include": ["in"]},
+      "matchnames": ["swaraj mazda", "sml isuzu"],
+      "tags": {
+        "brand": "SML Mahindra",
+        "brand:wikidata": "Q188514",
```

Comment:
Q188514 seems to be for Tata Motors.  Is this a mistake?
https://www.wikidata.org/wiki/Q188514